openstf/model/trainer/xgboost/xgboost.py

```diff
@@ -104,6 +109,7 @@ def train(
             validation_data (pandas.DataFrame): The validation data.
             callbacks (list of callable): List of callback functions that can be
                 called at the end of each training iteration
+            num_boost_round (int): can be large since we use early stopping.
```

Comment:
early stopping rounds seem to be missing in the docstring

Reply:
I'll include it!